internal/models/kubernetessecret/cluster/status.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package secret
+
+import (
+	"github.com/go-openapi/swag"
+	statusConditions "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/cluster/nodepool"
+)
+
+// VmwareTanzuManageV1alpha1ClusterNamespaceStatus Status of Secret resource.
+//
+// swagger:model vmware.tanzu.manage.v1alpha1.cluster.namespace.secret.Status
+type VmwareTanzuManageV1alpha1ClusterNamespaceStatus struct {
```

Comment:
If this is common across many TMC resources, please move it to a common folder/package.

Reply:
Updated PTAL